randovania/games/cave_story/exporter/hint_namer.py

```diff
@@ -64,6 +64,8 @@ def format_area(self, location: PickupLocation, with_region: bool, with_color: b
         return area.name
 
     def format_location_hint(self, game: RandovaniaGame, pick_hint: PickupHint, hint: Hint, with_color: bool) -> str:
+        assert hint.hint_type == HintType.LOCATION
```

Comment:
iirc @henriquegemignani wanted to have a fixme for this, but could be wrong.

Reply:
the fixme is on the `Hint` class